modules/account-lib/src/coin/sol/transactionBuilder.ts

```diff
@@ -13,8 +13,14 @@ import assert from 'assert';
 import { DurableNonceParams, InstructionParams, Memo, Nonce } from './iface';
 import { instructionParamsFactory } from './instructionParamsFactory';
 
+interface Signature {
```

Comment:
We can add this to `modules/account-lib/src/coin/baseCoin/iface.ts` since we use this in the `baseTransaction.ts` maybe?

Reply:
good call out. moving.

Reply:
Refactoring in subsequent PR.